Weather/Weather/Model/TypeInputData.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  TypeInputData.swift
+//  Weather
+//
+//  Created by Sergii Lisnyi on 5/18/18.
+//  Copyright © 2018 Sergii Lisnyi. All rights reserved.
+//
+
+import Foundation
+
+enum TypeInputData {
+    case location(latitude: String, longitude: String)
```

Comment:
I don't think you need to store data of different type. One data structure with city name and coordinates is ok.